_data/versions.yml

```diff
@@ -1,4 +1,54 @@
 v2.6:
+- title: v2.6.4-rc3
+  note: |
+    19 December 2017
+
+    This is a release candidate for testing purposes only.
+    v2.6.4-rc3 has a felix fix for "liveness" checking: https://github.com/projectcalico/felix/pull/1662
+
+    <div class="alert alert-danger" role="alert"><b>Important</b>: If you are using the Kubernetes datastore and upgrading from Calico v2.4.x or earlier to Calico v2.5.x or later, you must <a href="https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md">migrate your Calico configuration data</a> before upgrading. Otherwise, your cluster may lose connectivity after the upgrade.</div>
+
+  components:
+    felix:
+      version: 2.6.3-rc3
+      url: https://github.com/projectcalico/felix/releases/tag/2.6.3-rc3
+    typha:
+      version: v0.5.4-rc2
+      url: https://github.com/projectcalico/typha/releases/tag/v0.5.4-rc2
+    calicoctl:
+      version: v1.6.3-rc1
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.6.3-rc1
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.6.3-rc1/calicoctl
+    calico/node:
+      version: v2.6.4-rc3
+      url: https://github.com/projectcalico/calico/releases/tag/v2.6.4-rc3
+    calico/cni:
+      version: v1.11.2-rc1
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v1.11.2-rc1
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.11.2-rc1/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v1.11.2-rc1/calico-ipam
+    calico-bird:
+      version: v0.3.1
+      url: https://github.com/projectcalico/calico-bird/releases/tag/v0.3.1
+    confd:
+      version: v0.12.1-calico-0.4.3-rc1
+      url: https://github.com/projectcalico/confd/releases/tag/v0.12.1-calico-0.4.3-rc1
+    calico-bgp-daemon:
+      version: v0.2.1
+      url: https://github.com/projectcalico/calico-bgp-daemon/releases/tag/v0.2.1
+    libnetwork-plugin:
+      version: v1.1.2-rc1
+      url: https://github.com/projectcalico/libnetwork-plugin/releases/tag/v1.1.2-rc1
+    calico/kube-controllers:
+      version: v1.0.2-rc1
+      url: https://github.com/projectcalico/k8s-policy/releases/tag/v1.0.2-rc1
+    networking-calico:
+      version: 1.4.3
```

Comment:
I guess that this may not have been revved and so that's probably why it doesn't have an rc # (same as route reflector)...most likely intentional but just noting this.